pytorch/buildspec.yml

```diff
@@ -46,139 +46,79 @@ context:
       target: deep_learning_container.py
 
 images:
-  # BuildE3CPUPTTrainPy3DockerImage:
-  #   <<: *TRAINING_REPOSITORY
-  #   build: &PYTORCH_CPU_TRAINING_PY3 false
-  #   # enable_test_promotion: false
-  #   image_size_baseline: 5000
-  #   device_type: &DEVICE_TYPE cpu
-  #   python_version: &DOCKER_PYTHON_VERSION py3
-  #   tag_python_version: &TAG_PYTHON_VERSION py38
-  #   os_version: &OS_VERSION ubuntu20.04
-  #   tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-e3" ]
-  #   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
-  #   target: e3
-  #   context:
-  #     <<: *TRAINING_CONTEXT
-  # BuildE3GPUPTTrainPy3DockerImage:
-  #   <<: *TRAINING_REPOSITORY
-  #   build: &PYTORCH_GPU_TRAINING_PY3 false
-  #   # enable_test_promotion: false
-  #   image_size_baseline: 14000
-  #   device_type: &DEVICE_TYPE gpu
-  #   python_version: &DOCKER_PYTHON_VERSION py3
-  #   tag_python_version: &TAG_PYTHON_VERSION py38
-  #   cuda_version: &CUDA_VERSION cu113
-  #   os_version: &OS_VERSION ubuntu20.04
-  #   tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION, "-e3" ]
-  #   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile.,
-  #                        *DEVICE_TYPE ]
-  #   target: e3
-  #   context:
-  #     <<: *TRAINING_CONTEXT
-  BuildSageMakerCPUPTTrainPy3DockerImage:
+  BuildE3CPUPTTrainPy3DockerImage:
     <<: *TRAINING_REPOSITORY
     build: &PYTORCH_CPU_TRAINING_PY3 false
+    # enable_test_promotion: false
```

Comment:
what flag is this?

Reply:
This is a flag that allows contributors to "disable" testing on an image after it is built on CI, with no effect on pipeline tests. It seems to have been "added" by this PR, but it is actually just one of the uncommented lines from the e3 images defined in the buildspec from before this PR.

I can remove this flag in my next revision.